lib/csv/parser.rb

```diff
@@ -808,6 +798,13 @@ def adjust_headers(headers, quoted_fields)
 
     def prepare_parser
       @may_quoted = may_quoted?
+      @parse_method = if @quote_character.nil?
+        method(:parse_no_quote)
+      elsif @liberal_parsing || @strip
+        method(:parse_quotable_robust)
+      else
+        method(:parse_quotable_loose)
+      end
```

Comment:
Could you use the existing style?

```suggestion
      if @quote_character.nil?
        @parse_method = :parse_no_quote
      elsif @liberal_parsing or @strip
        @parse_method = :parse_quotable_robust
      else
        @parse_method = :parse_quotable_loose
      end
```

Reply:
Could you check this comment?

Reply:
Oh I missed `or` idea